examples/nim/default.nim

```diff
@@ -0,0 +1,3 @@
+# Type your code here, or load an example.
+proc square(num: int): int =
+    return num * num
```

Comment:
Just a nit, use 2 spaces and remove the `return` for more idiomatic Nim

Reply:
Something like this ?
```nim
proc square(i: int): int =  i * i
```

Reply:
```nim
proc square(num: int): int =
    result = num * num
```

```nim
proc square(num: int): int =
    num * num
```

Reply:
I meant 2-space indentation. Doing it on one line works as well,with just one space after the `=`, but is less friendly to modification if the user wants to add more code to the proc

Reply:
done!